src/objects/Account.object

```diff
@@ -0,0 +1,530 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I haven't seen any question that mentions editing Account object. Could you please verify?